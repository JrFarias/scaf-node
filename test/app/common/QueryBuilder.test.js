const QueryBuilder = require('../../../src/app/common/QueryBuilder');

jest.mock('../../../src/config/DB', (args) => args)

describe('QueryBuilder', () => {
  test('matches the snapshot', () => {
    const qb = jest.fn()
    const table = jest.fn()
    const queryBuilder = new QueryBuilder({ qb, table })

    expect(queryBuilder).toMatchSnapshot()
  })
})
