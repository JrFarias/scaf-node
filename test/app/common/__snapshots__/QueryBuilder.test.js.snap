// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QueryBuilder matches the snapshot 1`] = `
QueryBuilder {
  "qb": [MockFunction],
  "table": [MockFunction],
}
`;
